README.md

```
@@ -0,0 +1,102 @@
+	Point point = om.readValue(geoJsonString, Point.class);
+
+	// Or if the geometry type is unknown
+	Geometry geometry = om.readValue(geoJsonString, Geometry.class);
+}
+```
+
+### Serializing a LineString
+
+```java
+void serializeLineString() {
+	ArrayList<Position> coordinates = new ArrayList<>();
```

Markdownlint (0.37.0)

Column: 1
Hard tabs

(MD010, no-hard-tabs)